Esri/calcite-react
# fix(account manager): fix refresh and login functionality

## Description


These changes will fix bugs in sign-in behavior when adding multiple AGO/Enterprise accounts to allow the usage of query params to avoid redirects from '/authorize' endpoint. They also fix a bug when refreshing/ restoring Enterprise accounts which would always direct the user to an AGO login screen. 

## Related Issue





Closes #434 

## Motivation and Context


When attempting to add multiple accounts if user had directly authenticated with their org, '/authorize' woudl always redirect to '/approve' or back to the redirect_url. 

## How Has This Been Tested?




Test Redirect: (will need to add url to app to test in AGO or use client_Id with 'BprsHhCKRCOXoE86' for enterprise)
1. Login directly to AGO org
2. Attempt to add account when params is set
3. Verify no redirect

Test Enterprise Refresh/Restore: (replace client_Id with 'BprsHhCKRCOXoE86')
1. Login to Enterprise account
2. Attempt Restore
3. Attempt Refresh 

## Screenshots (if appropriate):

## Types of changes


## Checklist:
